notification.c

```diff
@@ -25,6 +25,26 @@ bool hotspot_at(struct mako_hotspot *hotspot, int32_t x, int32_t y) {
 		y < hotspot->y + hotspot->height;
 }
 
+void reset_notification(struct mako_notification *notif) {
```

Comment:
Hmm, this doesn't really reset, it frees everything inside the notification. To really reset it, would it be possible to set free'd pointers to NULL to make sure we don't use-after-free?

Reply:
Yep, good point.